### DISCUSSION TITLE
is it possible to call pip install of some python package from Julia code

### DISCUSSION THREAD

**Discussion by jakubMitura14** [cjdoris/PythonCall.jl#98] [Q&A] [answered]
is it possible to call pip install of some python package from Julia code? For Example I would like to use monai (not available on conda as far as I know).
Thanks !

**Comment by icweaver** [comment, marked as answer]
I think CondaPkg.jl uses the [conda-forge channel by default](https://github.com/cjdoris/CondaPkg.jl#faqs), which looks to have `monai`:

```julia
using PythonCall, CondaPkg

CondaPkg.add.("monai"); CondaPkg.resolve()

@py import monai

monai
# Python module: <module 'monai' from '/home/mango/Projects/scratch/.CondaPkg/env/lib/python3.9/site-packages/monai/__init__.py'>
``` 

And you can use `CondaPkg.add_pip` to use `pip` if you want, but I don't think it is [recommended](https://github.com/cjdoris/CondaPkg.jl#specify-dependencies-interactively)

**Reply by jakubMitura14** [reply]
Thanks!!
